docs/dev/dev_setup.rst

```diff
@@ -27,29 +25,44 @@ Linux) and are already familiar with ``git``, GitHub, and the Unix shell.
     * `Cloning a Repository <https://help.github.com/articles/cloning-a-repository/>`__
 
 ------------------------------------------------------------------------------
-Install ``conda`` / ``mamba``
+Install ``pixi``
 ------------------------------------------------------------------------------
-We use the ``mamba`` package manager to specify and update our development environment,
-preferentially installing packages from the community maintained `conda-forge
-<https://conda-forge.org>`__ distribution channel. We recommend using `miniforge
-<https://github.com/conda-forge/miniforge>`__ to install ``mamba`` and automatically
-default to the ``conda-forge`` channel.
+We use the ``pixi`` package manager to specify and update our development environment.
+Pixi uses the community maintained `conda-forge <https://conda-forge.org>`__
+distribution channel and provides reproducible environments through lockfiles.
 
-After installing your package manager, make sure it's configured to use
-`strict channel priority <https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-channels.html#>`__
-with the following commands:
+To install pixi, follow the instructions at https://pixi.sh or use one of these methods:
+
+**MacOS:**
+
+You can install pixi using Homebrew:
 
 .. code-block:: console
 
-    $ mamba update mamba
-    $ conda config --set channel_priority strict
+    $ brew install pixi
 
-.. note::
+Or use the installation script:
+
+.. code-block:: console
+
+    $ curl -fsSL https://pixi.sh/install.sh | bash
+
+**Linux:**
 
-   Make sure that after you've installed ``conda`` or ``mamba`` that you can activate
-   and deactivate the ``base`` environment (which is created by default). Try creating
-   a simple new environment and activating it to make sure everything is working and
-   your shell has been correctly configured to use the new package manager.
+.. code-block:: console
+
+    $ curl -fsSL https://pixi.sh/install.sh | bash
+
+After installation, restart your shell or source your profile to make the ``pixi``
+command available.
+
+**Windows:**
+
+The PUDL development environment is not currently supported on Windows because some
+dependencies are not available for Windows through conda-forge.  Windows users should
+use the `Windows Subsystem for Linux (WSL)
+<https://learn.microsoft.com/en-us/windows/wsl/install>`__ and follow the Linux
+installation instructions above.
```

Comment:
I think we should just point them at the pixi installation instructions and not vendor them here - if the install instructions change and we don't update this, it will just cause confusion.